MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs

```diff
@@ -77,12 +77,12 @@ private static IEnumerable<FileSystemMetadata> GetFiles(BaseItem item, bool incl
                 return directoryService.GetFileSystemEntries(path)
                 .Where(i => BaseItem.SupportedImageExtensions.Contains(i.Extension, StringComparer.OrdinalIgnoreCase) || i.IsDirectory)
 
-                .OrderBy(i => Array.IndexOf(BaseItem.SupportedImageExtensions, i.Extension ?? string.Empty));
+                .OrderBy(i => Array.IndexOf(BaseItem.SupportedImageExtensions.ToArray(), i.Extension ?? string.Empty));
```

Comment:
SupportedImageExtensions should be changed back to string[], not converted to array here

Reply:
#1071 is merged now, which should take care of this?